apps/docs/content/guides/local-development/declarative-database-schemas.mdx

```diff
@@ -7,7 +7,9 @@ subtitle: 'Manage your database schemas in one place and generate versioned migr
 
 ## Overview
 
-Declarative schemas allow you to reduce code duplications in managing [schema migrations](/docs/guides/local-development/overview#database-migrations). As your database schema evolves over time, declaring it in one place will help you iterate faster by referring to a single source of truth.
+Declarative schemas provide a developer-friendly way to maintain <InfoTooltip tooltipContent={<>Files of SQL statements that track the evolution of your database schema over time. They allow you to version control your database schema alongside your application code. See the <Link href="/guides/deployment/database-migrations" className="underline">database migrations</Link> guide to learn more.</>}>schema migrations</InfoTooltip>.
```

Comment:
possible to create a line-break in the tooltip?

<img width="1336" alt="image" src="https://github.com/user-attachments/assets/0697fbda-4bb7-4b91-88d5-11393ca29dba" />